build-windows/Get-Mailcore2.ps1

```diff
@@ -0,0 +1,40 @@
+Param(
+    [string]$InstallPath
+)
+
+$ProjectRoot = "$(Resolve-Path ""$PSScriptRoot\..\"")"
+
+if (-Not $InstallPath) {
+    $InstallPath = "$ProjectRoot\.build\install"
+}
+
+$PrebuiltMailcoreVersion = 1

```

Comment:
This is incrementally increasing version? Next will be 2?

Reply:
Yes, this is just arbitrary number to give us possibility to anchor different prebuilt variants. Not related to actual library version.

Reply:
Usually it is useful to have actual library version in file name, to easily answer a question "What version do we use?"

Reply:
It is not required for anchoring. For revision tracking it is useful to put a text file with revision info inside the archive, because it also contains some dependencies. This is exact same approach as mailcore uses for Darwin prebuilds.

I will check the cost of implementing such step in build script, but for me it is not a requirement now. The purpose of this task is to make build times a bit faster. The archive is built with current version of build script (with corresponding git revisions). If we will need to update archive, the build script would be updated accordingly.

Reply:
Thanks for detailed answer!